distribution/docs/src/main/resources/content/_developing/_devComponents/transformers-query-response.adoc

```diff
@@ -52,10 +51,6 @@ Query Response Transformers can be used through the Catalog transform convenienc
 |A short, human-readable description that describes the functionality of the service and the output.
 |_This service converts a single metacard xml document to an atom entry element_.
 
-|`export-resultset`
```

Comment:
✏️ Shouldn't `id` be included in this list of `service-properties` for `QueryResponseTransformer`s?